src/ngx_http_lua_kong_module.c

```diff
@@ -76,6 +84,7 @@ ngx_module_t ngx_http_lua_kong_module = {
 static ngx_int_t
 ngx_http_lua_kong_init(ngx_conf_t *cf)
 {
+    ngx_http_lua_kong_error_log_init(cf);
```

Comment:
We should check the return value of `ngx_http_lua_kong_error_log_init`.

Reply:
done, can you check again? thanks